bin/autodetect/CCmdLineParser.cc

```diff
@@ -107,19 +96,13 @@ bool CCmdLineParser::parse(int argc,
             ("persist", boost::program_options::value<std::string>(),
                     "Optional file to persist state to - not present means no state persistence")
             ("persistIsPipe", "Specified persist file is a named pipe")
-            ("persistInterval", boost::program_options::value<core_t::TTime>(),
-                    "Optional time interval at which to periodically persist model state (Mutually exclusive with bucketPersistInterval)")
             ("persistInForeground", "Persistence occurs in the foreground. Defaults to background persistence.")
             ("bucketPersistInterval", boost::program_options::value<std::size_t>(),
-                    "Optional number of buckets after which to periodically persist model state (Mutually exclusive with persistInterval)")
-            ("maxQuantileInterval", boost::program_options::value<core_t::TTime>(),
-                    "Optional interval at which to periodically output quantiles if they have not been output due to an anomaly - if not specified then quantiles will only be output following a big anomaly")
+                    "Optional number of buckets after which to periodically persist model state.")
```

Comment:
Is it now that if `bucketPersistInterval` is specified on the command line then it takes precedence over the persist interval in the config?

Reply:
Yes, we don't expect the `bucketPersistInterval` to ever be used in production code (it was introduced to support the `model_extractor` project). So if present it should take precedence over the background persist interval.